kotlinx-coroutines-test/common/src/TestCoroutineScheduler.kt

```diff
@@ -193,6 +193,14 @@ public class TestCoroutineScheduler : AbstractCoroutineContextElement(TestCorout
      * Consumes the knowledge that a dispatch event happened recently.
      */
     internal val onDispatchEvent: SelectClause1<Unit> get() = dispatchEvents.onReceive
+
+    /**
+     * Returns the [TimeSource] representation of the virtual time of this scheduler.
+     */
+    @ExperimentalTime
```

Comment:
Maybe it is worth marking it as `ExperimentalCoroutinesApi` as well in case we'll decide to implement `TimeSource` interface when it's stable?

Reply:
Good point!

Reply:
Sure